apps/web/pages/video/[uid].tsx

```diff
@@ -51,6 +63,14 @@ export default function JoinCall(props: JoinCallPageProps) {
           supportiveText: "#FFF",
         },
       },
+      customTrayButtons: {
+        transcriptButton: {
+          iconPath: "https://unpkg.com/lucide-static@latest/icons/sparkles.svg", // TODO
+          iconPathDarkMode: "https://unpkg.com/lucide-static@latest/icons/sparkles.svg", //TODO
```

Comment:
i think we can move these into our public/ folder

Reply:
we need to use the url.
We can fix the css by adding styles for `#transcriptButton-controls`